Core/GDCore/IDE/WholeProjectRefactorer.cpp

```diff
@@ -393,8 +384,18 @@ void WholeProjectRefactorer::RenameBehaviorEventsFunction(
   const gd::EventsFunction& eventsFunction =
       eventsFunctions.GetEventsFunction(oldFunctionName);
 
-  if (eventsFunction.GetFunctionType() == gd::EventsFunction::Action ||
-      eventsFunction.GetFunctionType() == gd::EventsFunction::Condition) {
+  // The order is important
```

Comment:
I'm not exactly sure why. So I didn't give details, but with the other way around, expressions inside their own condition weren't renamed.

Reply:
Ah I just saw this comment. Ok then!